config.json

```diff
@@ -14,6 +14,18 @@
         "initiateLifetime": {
             "hours": 1
         },
+        "loginDir": "/login/",
+        "loginHash": "sha512",
+        "loginIdLength": 24,
+        "loginLifetime": {
+            "minutes": 15
+        },
+        "passwordHash": {
+            "algo": "sha512",
+            "hashLength": 48,
```

Comment:
This is an ambiguous number.  Is that the length of the resulting HASH?  No.  It is the length of the resulting bytes that are generated from pbkdf2.  This number can be anything, where as a hash is a fixed length.  Maybe investigate pbkdf2 implementations in different languages and see what standard there is for the name of the length of the result.